### PR TITLE
DTTSB-2179 (timeline component updates)

### PR DESCRIPTION
**Pull request by planctus**
The pull request needs to be evaluated together with these additional PR:
https://github.com/ec-europa/digital-transformation-dev/pull/1620
https://github.com/ec-europa/ec-europa-theme-tools/pull/108

### CODE REVIEW

**Comment by nelsongrin** [comment]
Hi @planctus 
Why did you changed the margins in the component?
When I delivered the work about the timeline I respected the instructions from Filipa.
Did you noticed something wrong then?

**Comment by nelsongrin** [comment]
Why not open this against release/1.23?

**Comment by planctus** [comment]
@nelsongrin: style changes are minimal, those were needed fixes in the different implementations we have at the moment. 
Let's use develop while developing, when we are about to release we will move to the feature branch. I merged in develop what was in the 1.23, i didn't delete the branch and we can keep maintaining that or just dropping it and recreating that in the next weeks.
I've updated the composer.json in order to use the custom branches from theme and tools. Don't forget to change it back when merging, thanks.

File: sass/components/_timeline.scss [hunk 1]
```diff
@@ -8,11 +8,12 @@ Style guide timeline
 
 .timeline {
   border-left: 4px solid $ec-yellow;
+  margin-bottom: 1.667em;
 
   .btn-timeline {
     @extend .btn-ctn;
 
-    margin: 0 0 0 1.667em;
+    margin: 0 0 1.667em 1.667em;
```
Comment (nelsongrin) [inline sass/components/_timeline.scss]: Change this to margin: 0 1.667em;

**Pull request merged**